module-services/service-appmgr/model/ApplicationManagerCommon.cpp

```diff
@@ -157,14 +157,12 @@ namespace app::manager
         ActionRequest act = ActionRequest{this->GetName(), app::manager::actions::DisplayLogoAtExit, nullptr};
         switch (closeReason) {
         case sys::CloseReason::SystemBrownout:
-            [[fallthrough]];
```

Comment:
Doesn't the compiler complain about the lack of `fallthrough`?

Reply:
No, it is designed for slightly different case https://en.cppreference.com/w/cpp/language/attributes/fallthrough